traits/traits_listener.py

```diff
@@ -837,24 +818,13 @@ def _get_value(self, name):
 
 class ListenerGroup(ListenerBase):
 
-    #: Does the handler go at the beginning (True) or end (False) of the
-    #: notification handlers list?
-    priority = ListProperty
-
     #: The next level (if any) of ListenerBase object to be called when any of
     #: this object's listened-to traits is changed
     next = ListProperty
```

Comment:
I don't know if it'll be useful performance-wise to get rid of the `ListProperty` traits here - `next` and `notify`.

Reply:
I'm looking at that: it would be necessary if we want to make these plain old Python classes instead of `HasTraits` classes. But it's not completely trivial, so I plan to defer that to another PR.